### DISCUSSION TITLE
Can Poiji read the formula value of the cells instead the value within the cell itself?

### DISCUSSION THREAD

**Discussion by Ajeky** [ozlerhakan/poiji#199] [Q&A]
I'm currently working on an application that reads a lot of monetary values from excel files. One would think that, as monetary values, having Excel round the values to 2 decimals would be fine, but In my case there are a lot of values with 3 or even more decimal digits that I can't afford to trim or round since the loss of information would be catastrophic. 

The current solution I have is to store the values as text instead of numeric to circunvent the automatic rounding that Excels perform in numeric cells, but that brings a lot of other problems to the table that are pretty annoying to fix. 

I was wondering if it would be possible to read the value from the formula itself instead of from the cell, since the value from the formula contains the whole number instead of the rounded value, making it a much optimal solution for my situation.

Thanks beforehand and thanks for the nice tool you offer!

**Comment by ozlerhakan** [comment]
Hi @Ajeky ,

Sorry for the late response.  Does [feature-13](https://github.com/ozlerhakan/poiji#feature-13) work for you?

or does it relate with https://github.com/ozlerhakan/poiji/issues/138 ?
